packages/modules/link-modules/src/definitions/order-return-fulfillment.ts

```diff
@@ -0,0 +1,67 @@
+import { ModuleJoinerConfig } from "@medusajs/types"
+import { LINKS, Modules } from "@medusajs/utils"
+
+export const ReturnFulfillment: ModuleJoinerConfig = {
+  serviceName: LINKS.ReturnFulfillment,
+  isLink: true,
+  databaseConfig: {
+    tableName: "return_fulfillment",
+    idPrefix: "retful",
+  },
+  alias: [
+    {
+      name: ["return_fulfillment", "return_fulfillments"],
+      args: {
+        entity: "LinkReturnFulfillment",
+      },
+    },
+  ],
+  primaryKeys: ["id", "return_id", "fulfillment_id"],
+  relationships: [
+    {
+      serviceName: Modules.ORDER,
+      primaryKey: "id",
+      foreignKey: "return_id",
+      alias: "return",
+      args: {
+        methodSuffix: "Returns",
+      },
+    },
+    {
+      serviceName: Modules.FULFILLMENT,
+      primaryKey: "id",
+      foreignKey: "fulfillment_id",
+      alias: "fulfillments",
+      args: {
+        methodSuffix: "Fulfillments",
+      },
+    },
+  ],
+  extends: [
+    {
+      serviceName: Modules.ORDER,
+      fieldAlias: {
+        return_fulfillments: {
```

Comment:
thought: I wasn't expecting returns to have multiple fulfillments. What's your thinking?

Reply:
That's just how it is desined on the backend.
in theory it could be split in multiple shipments, even if we have many places where multiple shipping options are not handled properly.
Should I just remove the `isList`?

Reply:
Got it. If it doesn't increase complexity to have multiple, then I would be fine with having `isList` for now 👍

Reply:
current there is an issue expanding this link, I will investigate it later and possibly push a fix in another PR.

Reply:
At the end, we need the shipping option from the shipping method created, so the location_id of the return at this point is not super relevant.
Unless we want to validate that maybe?
if the shipping_option of the shipping method is in a different location_id of the return